Lab2/scripts/deployment.sh

```diff
@@ -60,12 +71,23 @@ if [[ $server -eq 1 ]]; then
   fi
 
   sam build -t template.yaml --use-container
-  sam deploy --config-file samconfig.toml --region=$REGION
+
+  if [ "$IS_RUNNING_IN_EVENT_ENGINE" = true ]; then
+    sam deploy --config-file samconfig.toml --region=$REGION --parameter-overrides EventEngineParameter=$IS_RUNNING_IN_EVENT_ENGINE AdminUserPoolCallbackURLParameter=$ADMIN_SITE_URL
```

Comment:
line 76 and 78 are duplicate

Reply:
The line 76 has one additional parameter "AdminUserPoolCallbackURLParameter"